From 5f8d44eb722a228e8cc8c51a8378a53837bb81c9 Mon Sep 17 00:00:00 2001
From: Ayoub Zaki <ayoub.zaki@embetrix.com>
Date: Mon, 15 Dec 2025 13:32:50 +0100
Subject: [PATCH] bitcoind: disable noisy UpdateTipLog logs

Signed-off-by: Ayoub Zaki <ayoub.zaki@embetrix.com>
---
 src/validation.cpp | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/validation.cpp b/src/validation.cpp
index 3f77955da6..4eab242694 100644
--- a/src/validation.cpp
+++ b/src/validation.cpp
@@ -2941,15 +2941,15 @@ static void UpdateTipLog(
     AssertLockHeld(::cs_main);
 
     // Disable rate limiting in LogPrintLevel_ so this source location may log during IBD.
-    LogPrintLevel_(BCLog::LogFlags::ALL, BCLog::Level::Info, /*should_ratelimit=*/false, "%s%s: new best=%s height=%d version=0x%08x log2_work=%f tx=%lu date='%s' progress=%f cache=%.1fMiB(%utxo)%s\n",
-                   prefix, func_name,
-                   tip->GetBlockHash().ToString(), tip->nHeight, tip->nVersion,
-                   log(tip->nChainWork.getdouble()) / log(2.0), tip->m_chain_tx_count,
-                   FormatISO8601DateTime(tip->GetBlockTime()),
-                   chainman.GuessVerificationProgress(tip),
-                   coins_tip.DynamicMemoryUsage() * (1.0 / (1 << 20)),
-                   coins_tip.GetCacheSize(),
-                   !warning_messages.empty() ? strprintf(" warning='%s'", warning_messages) : "");
+    // LogPrintLevel_(BCLog::LogFlags::ALL, BCLog::Level::Info, /*should_ratelimit=*/false, "%s%s: new best=%s height=%d version=0x%08x log2_work=%f tx=%lu date='%s' progress=%f cache=%.1fMiB(%utxo)%s\n",
+    //                prefix, func_name,
+    //                tip->GetBlockHash().ToString(), tip->nHeight, tip->nVersion,
+    //                log(tip->nChainWork.getdouble()) / log(2.0), tip->m_chain_tx_count,
+    //                FormatISO8601DateTime(tip->GetBlockTime()),
+    //                chainman.GuessVerificationProgress(tip),
+    //                coins_tip.DynamicMemoryUsage() * (1.0 / (1 << 20)),
+    //                coins_tip.GetCacheSize(),
+    //                !warning_messages.empty() ? strprintf(" warning='%s'", warning_messages) : "");
 }
 
 void Chainstate::UpdateTip(const CBlockIndex* pindexNew)
-- 
2.43.0

